images/win/scripts/Tests/NodeLTS.Tests.ps1

```diff
@@ -0,0 +1,13 @@
+Describe "Node LTS" {
+    It "Checking <ToolName> " -TestCases @(
```

Comment:
I don't think that `It "Checking <ToolName> "` is good naming for tests.
It is not debug print so I suggest to name them like a tool
`It "<ToolName>"`. If it is more complex test, let's describe details. The same for the whole PR

Reply:
Thank you, done.

Reply:
Could you please adjust this comment to the whole PR?

Reply:
Done